cernopendata/modules/fixtures/data/docs/cms-guide-for-research/cms-guide-for-research.md

```diff
@@ -1,213 +1,228 @@
+      <th>CMSSW version</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2010&f=collision_energy%3A7TeV&l=list&order=desc&p=1&s=10&sort=mostrecent">proton-proton 2010</a></td>
+      <td>7</td>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=year%3A2010&f=collision_energy%3A7TeV&l=list&order=desc&p=1&s=10&sort=mostrecent">2010 simulation</a></td>
+      <td><a href="/docs/cms-getting-started-aod">AOD data</a></td>
+      <td>CMSSW_4_2_8</td>
+    </tr>
+    <tr>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2011&f=collision_energy%3A7TeV&l=list&order=desc&p=1&s=10&sort=mostrecent">proton-proton 2011</a></td>
+      <td>7</td>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=collision_energy%3A7TeV&f=year%3A2011&l=list&order=desc&p=1&s=10&sort=mostrecent">2011 simulation</a></td>
+      <td><a href="/docs/cms-getting-started-aod">AOD data</a></td>
+      <td>CMSSW_5_3_32</td>
+    </tr>
+    <tr>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2012&l=list&order=desc&p=1&s=10&sort=mostrecent">proton-proton 2012</a></td>
+      <td>8</td>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=year%3A2012&l=list&order=desc&p=1&s=10&sort=mostrecent">2012 simulation</a></td>
+      <td><a href="/docs/cms-getting-started-aod">AOD data</a></td>
+      <td>CMSSW_5_3_32</td>
+    </tr>
+    <tr>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2015&f=collision_energy%3A13TeV&l=list&order=desc&p=1&s=10&sort=mostrecent">proton-proton 2015</a></td>
+      <td>13</td>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=year%3A2015&l=list&order=desc&p=1&s=10&sort=mostrecent">2015 simulation</a></td>
+      <td><a href="/docs/cms-getting-started-miniaod">MiniAOD data</a></td>
+      <td>CMSSW_7_6_7</td>
+    </tr>
+    <tr>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2016&l=list&order=desc&p=1&s=10&sort=mostrecent">proton-proton 2016</a></td>
+      <td>13</td>
+      <td><a href="https://opendata.cern.ch/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=year%3A2016&l=list&order=desc&p=1&s=10&sort=mostrecent">2016 simulation</a></td>
+      <td><a href="/docs/cms-getting-started-miniaod">MiniAOD data</a><br><a href="/docs/cms-getting-started-nanoaod">NanoAOD data</a></td>
+      <td>CMSSW_10_6_30<br>Not required</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
@psaiz Here as well, is there an easy way to add space between the columns?
![image](https://github.com/cernopendata/opendata.cern.ch/assets/5859918/2cfe696c-1096-4523-9646-9d9cfd064a87)